### ISSUE TITLE
ImportError : cannot import name 'TodoBlock' from 'notion.block'

### ISSUE THREAD

**Issue by adityakarad17** [jamalex/notion-py#321]
```
Traceback (most recent call last):
  File "main.py", line 55, in <module>
    from notion.block import TodoBlock
ImportError: cannot import name 'TodoBlock' from 'notion.block' (/opt/virtualenvs/python3/lib/python3.8/site-packages/notion/block/__init__.py)
```

**Comment by losnah-think** [comment]
How did you solve it?
I'm in the same situation.

``` shell
Traceback (most recent call last):
  File ".\write_notion.py", line 4, in <module>
    from notion.block import TodoBlock
ImportError: cannot import name 'TodoBlock' from 'notion.block' (...\Python\Python38\site-packages\notion\block\__init__.py)
```

**Comment by adityakarad17** [comment]
I had this error on Ubuntu. But when I switched over to Windows, it was fine. I figured out that the Python libraries on Ubuntu had some issue. I re-installed Python, then it worked fine. Just remove python with care since I've read about it breaking Ubuntu completely.

**Comment by losnah-think** [comment]
I ran PYTHON in the Window environment. I thought it was an environmental problem, so the same problem occurred even if I changed it to MAC environment. I will proceed by downgrading the version of the notion-py library. If I solve this problem, I will leave a reply below.

**Comment by losnah-think** [comment]
I just solved the problem that occurred.
When I installed the 'notion-py' library, I thought it would be compatible to the 'notion' library, so I didn't proceed with the installation separately.
And after various attempts, we found that the problem disappeared while installing the 'notion' library in version 0.0.25.
I'll save it for other people who might see it.

If you're having the same problem as I am, try the method below.
``` shell
pip3 install "notion === 0.0.25"
```

@adityakarad17  Thank you for helping me. I got a hint from what you said.
